river/tree/_nodes/efdtc_nodes.py

```diff
@@ -85,7 +76,7 @@ def best_split_suggestions(self, criterion, tree):
         best_suggestions = []
         pre_split_dist = self.stats
 
-        for idx, obs in self.attribute_observers.items():
+        for idx, obs in self.splitters.items():
```

Comment:
Forgot to change to `splitter`. I will do that later.

Reply:
Changed in commit [d6931d9](https://github.com/online-ml/river/pull/488/commits/d6931d9e1632978aeb7276b9a13cff1ac2a3ec09).